index.html

```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" ><div class ="otherproject">
+            <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+            <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul id ="project2language">
+                <li class ="language2item">html</li>
+                <li class ="language2item" id ="bootstrap">bootstrap</li>
+                <li class ="language2item">Ruby</li>
+            </ul>
+            <button class="seeproject2">See Project</button>
+        </div>
+    </section>
+        
+    </div>
+    <div class="aboutsection" >
+        <section class ="about">
 
+            <h1 class="aboutheader"> About me</h1>
```

Comment:
- [ ] Please refer to my previous comment on multiple `h1` tags on a webpage. Kindly replace this `h1` with another header tag, maybe `h2`.